src/tests/ftest/util/server_utils.py

```diff
@@ -467,9 +467,9 @@ def support_collect_log(self, **kwargs):
         cmd = DaosServerCommand(self.manager.job.command_path)
         cmd.run_user = "daos_server"
         cmd.debug.value = False
-        cmd.config.value = get_default_config_file("server")
+        cmd.set_command(("support", "collect-log"),
+                        "config"=get_default_config_file("server"), **kwargs)
```

Comment:
As we discussed:
```
        kwargs['config'] = get_default_config_file("server")
        cmd.set_command(("support", "collect-log"), **kwargs)
```

Reply:
as Phil points out, there is a subtle difference in the case where the config value is set in kwargs and supplied in set_command parameters